Sources/BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -339,7 +339,7 @@ public class CryptoStore: ObservableObject {
   func prepare(isInitialOpen: Bool = false) {
     Task { @MainActor in
       if isInitialOpen {
-        portfolioStore.discoverAssetsOnAllSupportedChains()
```

Comment:
Missed this but `discoverAssetsOnAllSupportedChains()` function can be removed from `PortfolioStore` now 🙂.

Reply:
updated in [1f36354](https://github.com/brave/brave-ios/pull/7400/commits/1f36354a1f6371547642dfbc68c81d3639fe4af8)